api/src/main/java/io/grpc/ServerProvider.java

```diff
@@ -34,28 +33,13 @@
  */
 @Internal
 public abstract class ServerProvider {
-  private static final ServerProvider provider = ServiceProviders.load(
-      ServerProvider.class,
-      Collections.<Class<?>>emptyList(),
-      ServerProvider.class.getClassLoader(),
-      new PriorityAccessor<ServerProvider>() {
-        @Override
-        public boolean isAvailable(ServerProvider provider) {
-          return provider.isAvailable();
-        }
-
-        @Override
-        public int getPriority(ServerProvider provider) {
-          return provider.priority();
-        }
-      });
-
   /**
    * Returns the ClassLoader-wide default server.
```

Comment:
Not part of this PR but would "... default server provider." be more accurate?

Reply:
Yes, although "server" doesn't have to mean the server _class_. It can simply mean something like "netty" here (like you'd say "apache" or "nginx"). The ServerProvider is how you'd use that server.